src/plugins/postgres_permissions_operator.py

```diff
@@ -117,6 +133,11 @@ def execute(self, context=None):  # noqa: C901
 
                 for dataset_name in executed_dags_after_delta:
 
+                    # get real datasetname from DAG_DATASET constant, if dag_id != dataschema name
+                    for key in DAG_DATASET.keys():
+                        if key in dataset_name:
+                            dataset_name = DAG_DATASET[key]
+
```

Comment:
You might want to add a `break` statement here to terminate the `for` loop once you've found the key. Without a `break` statement this `for` loop could be interpreted as _find the last matching key_, whereas there should only ever be one match.

```python
dataset_name = DAG_DATASET[key]
break
```

Reply:
Goeie, zal ik aanpassen